game-app/game-core/src/main/java/tools/image/DecorationPlacer.java

```diff
@@ -779,7 +780,8 @@ private void fillCurrentImagePointsBasedOnImageFolder(
                   ? ImagePointType.SPACE_BETWEEN_NAMES_AND_PUS
                   : 0));
       final List<String> allTerritories = new ArrayList<>(centers.keySet());
-      for (final File file : FileUtils.listFiles(currentImageFolderLocation)) {
+      for (final Path path : FileUtils.listFiles(currentImageFolderLocation)) {
+        final File file = path.toFile();
         if (!file.getPath().endsWith(".png") && !file.getPath().endsWith(".gif")) {
```

Comment:
This would be a good place to use FileUtils IMO, the conversion from path to file, back to path seems wonky. Do we even need to convert back to file here at all? Immediately we have a path, goes to a file, then goes back to a path again.

Reply:
There is no need to convert it back, but note that `getPath()` != `toPath()`, the former returns a string, the latter a `java.nio.Path` object.
After doing some experiments it seems like `File#getPath` is the equivalent of `Path#toString` so I could easily replace that when I'm thinking about it